apps/omg_utils/mix.exs

```diff
@@ -25,5 +25,13 @@ defmodule Utils.MixProject do
   defp elixirc_paths(:prod), do: ["lib"]
   defp elixirc_paths(_), do: ["lib", "test/support"]
 
-  defp deps, do: [{:statix, "~> 1.1"}, {:appsignal, "~> 1.9"}]
+  defp deps do
+    [
+      {:appsignal, "~> 1.9"},
+      {:statix, "~> 1.1"},
+      {:spandex, "~> 2.4"},
+      {:spandex_datadog, "~> 0.4"},
```

Comment:
I see `:spandex` has Plugs https://github.com/spandex-project/spandex#phoenix-plugs
what are we getting from `:spandex_datadog`?